designer/client/fee-items.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { clone } from './helpers'
+import { clone } from 'digital-form-builder-model/lib/helpers'
```

Comment:
i'll stop mentioning `see comment for component-condition-create.js` but you get the idea..!